### ISSUE TITLE
Cannot launch debug target (spawn node ENOENT).

### ISSUE THREAD

**Issue by korzio** [Microsoft/vscode#7305]
- VSCode Version: 1.2.0 (1.2.0)
- OS Version: MacOS 10.11.4 (15E65)

Steps to Reproduce:

1. Create launch.json profile
f.e.
{
      "name": "Launch",
      "type": "node",
      "request": "launch",
      "program": "${workspaceRoot}/a.js",
      "stopOnEntry": false,
      "args": [],
      "cwd": "${workspaceRoot}",
      "preLaunchTask": null,
      "runtimeExecutable": null,
      "runtimeArgs": [
        "--nolazy"
      ],
      "env": {
        "NODE_ENV": "development"
      },
      "externalConsole": false,
      "sourceMaps": false,
      "outDir": null
    },
2. Create a.js in a root of a project
3. Run Debug
4. It throws an error "Cannot launch debug target (spawn node ENOENT)."

**Comment by korzio** [comment]
workbench.main.js:87 Cannot launch debug target (spawn node ENOENT).: Error: Cannot launch debug target (spawn node ENOENT).
    at Object.E [as create] (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:31:17647)
    at file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:104:23498
    at Object.p [as _notify] (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:16345)
    at Object.__dirname.undefined.F.enter (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:19673)
    at __dirname.undefined.t.Class.define._run (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:20994)
    at __dirname.undefined.t.Class.define._chainedError (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:20353)
    at n (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:18181)
    at Object.p [as _notify] (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:16345)
    at Object.__dirname.undefined.F.enter (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:19673)
    at __dirname.undefined.t.Class.define._run (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:20994)





--------------



messageService.ts:119Error: spawn EACCES
    at exports._errnoException (util.js:890:11)
    at ChildProcess.spawn (internal/child_process.js:302:11)
    at Object.exports.spawn (child_process.js:379:9)
    at NodeDebugSession.launchRequest (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node-debug/out/node/nodeDebug.js:457:34)
    at NodeDebugSession.DebugSession.dispatchRequest (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node-debug/node_modules/vscode-debugadapter/lib/debugSession.js:303:22)
    at NodeDebugSession.ProtocolServer._handleData (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node-debug/node_modules/vscode-debugadapter/lib/protocol.js:78:38)
    at Socket.<anonymous> (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node-debug/node_modules/vscode-debugadapter/lib/protocol.js:23:60)
    at emitOne (events.js:90:13)
    at Socket.emit (events.js:182:7)
    at readableAddChunk (_stream_readable.js:153:18): Error: Error: spawn EACCES
    at exports._errnoException (util.js:890:11)
    at ChildProcess.spawn (internal/child_process.js:302:11)
    at Object.exports.spawn (child_process.js:379:9)
    at NodeDebugSession.launchRequest (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node-debug/out/node/nodeDebug.js:457:34)
    at NodeDebugSession.DebugSession.dispatchRequest (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node-debug/node_modules/vscode-debugadapter/lib/debugSession.js:303:22)
    at NodeDebugSession.ProtocolServer._handleData (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node-debug/node_modules/vscode-debugadapter/lib/protocol.js:78:38)
    at Socket.<anonymous> (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/node-debug/node_modules/vscode-debugadapter/lib/protocol.js:23:60)
    at emitOne (events.js:90:13)
    at Socket.emit (events.js:182:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Object.E [as create] (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:31:17647)
    at file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:104:23498
    at Object.p [as _notify] (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:16345)
    at Object.__dirname.undefined.F.enter (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:19673)
    at __dirname.undefined.t.Class.define._run (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:20994)
    at __dirname.undefined.t.Class.define._chainedError (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:20353)
    at n (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:18181)
    at Object.p [as _notify] (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:16345)
    at Object.__dirname.undefined.F.enter (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:19673)
    at __dirname.undefined.t.Class.define._run (file:////Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:23:20994)

**Comment by korzio** [comment]
So, this helped:

- Reinstall VSCode
- Reinstall node

-------------

I had a problem with node versions. 
I've used node v5.* in a system, while local version from /usr/local/bin/node -v
v4.4.5

So, VSCode was using v4.4.5, and I could not change it in settings, so I've removed a link from usr/local/bin catalog. And it was using v5 as a default for some time. But after update to a new VSCode version it started to throw an ENOENT error (as in title).

And this inconsistency in versions were preventing me to debug a project due to sass-package, build with other node version
```
".../node-sass/vendor/darwin-x64-46/binding.node" exists. 
 testing binary.
Binary is fine; exiting.
```

So, it is a bit complicated :)
Anyway, I think, there should be something improved I guess :)
